apps/admin_app/lib/admin_app_web/router.ex

```diff
@@ -31,6 +31,9 @@ defmodule AdminAppWeb.Router do
     # Use the default browser stack
     pipe_through([:browser, :authentication])
 
+    get("/", PageController, :index)
```

Comment:
@jyotigautam can you give a brief explanation about this for me? Thanks!

Reply:
@jyotigautam Can we remove this route as this route has been moved to [line 110](https://github.com/aviacommerce/avia/blob/c5c274762c73caa6e16d7a8e4259240cab237d0a/apps/admin_app/lib/admin_app_web/router.ex#L110) in [PR 375 - React integration](https://github.com/aviacommerce/avia/pull/375).

This will break the react integration and the request will not be serverd properly by react.